MoreLinq.Test/ToDelimitedStringTest.cs

```diff
@@ -32,7 +32,7 @@ public void ToDelimitedStringWithNullSequence()
             MoreEnumerable.ToDelimitedString<int>(null, ",");
         }
 
-        [Test]
+        [Test, Ignore("Obsolete")]
```

Comment:
Why mark the tests as obsolete? I would leave them intact because as long as the methods are part of 2.x (albeit being marked obsolete), the tests would make sure that no one breaks them by accident through any sort of refactoring.